pom.xml

```
@@ -47,8 +44,8 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <copyrightYear>2022</copyrightYear>
```

```suggestion
    <copyrightYear>2022</copyrightYear>
    <copyrightNotice>Copyright ${project.inceptionYear}-${copyrightYear} Micro Focus or one of its affiliates.</copyrightNotice>
```
This needs added if inceptionYear and copyright Year are the same. Also copyrightYear needs updated to 2023.
Alternatively if inceptionYear is 2022 and copyrightYear is 2023 then the notice is not needed.